shared/wallets/send-form/header.js

```diff
@@ -24,22 +24,35 @@ const styles = styleSheetCreate({
     left: 16,
     position: 'absolute',
   },
-  header: {
-    alignSelf: 'flex-end',
-    backgroundColor: globalColors.purple,
-    borderTopLeftRadius: 4,
-    borderTopRightRadius: 4,
-    height: 48,
-    justifyContent: 'center',
-    position: 'relative',
-  },
-  headerContainer: {
-    height: 60,
-  },
-  icon: {
-    position: 'relative',
-    top: -9,
-  },
+  header: platformStyles({
```

Comment:
I had to change a couple of styles in the `SendForm` header to make it work with `MaybePopup` and get that coins-hanging-over-the-edge look. When testing on mobile, I noticed the `SendForm` styles didn't seem to be fully done. I didn't want to get fully into writing those in this PR, but I tried to add some `platformStyles` to make things a little nicer.